src/components/ComponentSettingsModal.jsx

```diff
@@ -178,6 +178,7 @@ const ComponentSettingsModal = ({
         const frontMatter = isPost 
           ? { ...originalFrontMatter, title, date: resourceDate, permalink }
           : { ...originalFrontMatter, title, date: resourceDate, file_url: fileUrl }
+        if (isNewFile && isPost) frontMatter.layout = 'post'
```

Comment:
Just to keep things clean, can we subsume this line into line 179, as in
`{ ...originalFrontMatter, title, date: resourceDate, permalink, layout: post }`

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/478/commits/e31ae39d3045482f94fe131d11cd484b7ad627ee. As discussed offline, we should also check if users use custom layouts in resources, since this would replace their custom layout when editing page settings.